### PR TITLE
Adding Jacey

### PR DESCRIPTION
**Pull request by jaceybronte**
Adding Jacey's bio and image to the website

### CODE REVIEW

**Comment by jaceybronte** [comment]
@gwaybio

File: _members/jacey-curd.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - Julia B Curd
+    - Julia Bronte Curd
+    - Jacey Bronte
+    
+links:
+  email: julia.curd@cuanschutz.edu
+  github: JaceyBronte
+---
+
+Jacey graduated with B.S. degrees in Computer Science and Biology from Duke University, where she worked in Dr. Raluca Gordan’s lab on CRISPR and MutS. In her senior thesis, she focused on the DNA binding specificity of MutS, the primary enzyme responsible for initiating mismatch repair, and possible associations with CRISPR Prime Editing technology, specifically in single base pair edits.
```
Comment (gwaybio) [inline _members/jacey-curd.md:17]:
```suggestion
Jacey graduated with B.S. degrees in Computer Science and Biology from Duke University, where she worked in Dr. Raluca Gordan’s lab on CRISPR and MutS.
In her senior thesis, she focused on the DNA binding specificity of MutS, the primary enzyme responsible for initiating mismatch repair, and possible associations with CRISPR Prime Editing technology, specifically in single base pair edits. 
```

We write [one-sentence-per-line](https://news.ycombinator.com/item?id=31808093) in markdown format.

File: _members/jacey-curd.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+links:
+  email: julia.curd@cuanschutz.edu
+  github: JaceyBronte
+---
+
+Jacey graduated with B.S. degrees in Computer Science and Biology from Duke University, where she worked in Dr. Raluca Gordan’s lab on CRISPR and MutS. In her senior thesis, she focused on the DNA binding specificity of MutS, the primary enzyme responsible for initiating mismatch repair, and possible associations with CRISPR Prime Editing technology, specifically in single base pair edits. 
+
+Jacey joined the Way Lab in July 2024, and is incredibly passionate about researching pediatric oncology, with a focus on computational methods. 
+
+Outside of the lab, Jacey enjoys ballet, salsa, and line dancing, and also helps mentor a FIRST robotics team.
```
Comment (gwaybio) [inline _members/jacey-curd.md:21]: is FIRST an acronym? You might consider writing this out if so!

File: _members/jacey-curd.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+name: Jacey Curd
+image: images/team/jacey_curd.jpeg
+description: Bio Data Scientist
```
Comment (gwaybio) [inline _members/jacey-curd.md:4]:
```suggestion
description: Data Scientist
```

To align with Cameron (on the website at least!)

**Review by gwaybio — APPROVED** [review]
I made a few suggestions, otherwise LGTM! (all links work in the deploy preview)

Please address the suggestions and then squash + merge

**Comment by jaceybronte** [comment]
@gwaybio edits made!

**Comment by gwaybio** [comment]
Looks good! Are you able to squash and merge? Or do you not have access?

**Comment by jaceybronte** [comment]
@gwaybio I don't believe I have access

**Comment by gwaybio** [comment]
I should have just given you access - try reloading!

**Pull request merged**